pkg/config/config.go

```diff
@@ -580,6 +584,7 @@ func DefaultConfig() (*Config, error) {
 			HooksDir:                 []string{hooks.DefaultDir},
 			NamespacesDir:            "/var/run",
 			ManageNSLifecycle:        true,
+			DropInfraCtr:             false,
```

Comment:
afaik you don't have to set a value if it's a default one for this type (for boolean the default is false).

Reply:
...unless you're doing it for readability.